en/components/radial-gauge.md

```diff
@@ -24,12 +24,22 @@ The Radial Gauge provides various overrides that control the shape of the needle
 
 ## Dial Type
 
-The Radial Gauge provides various overrides that control the dial, on which ranges appear: **full**, half, inside, and outside.
+The Radial Gauge provides various overrides that control the dial, on which ranges appear: **full**, half, quarter with thick and thin overrides.
 
 <img class="responsive-img" src="../images/radial_gauge_three_ranges.png" srcset="../images/radial_gauge_three_ranges@2x.png 2x" />
 <img class="responsive-img" src="../images/radial_gauge_half.png" srcset="../images/radial_gauge_half@2x.png 2x" />
+<img class="responsive-img" src="../images/radial_gauge_quarter.png" srcset="../images/radial_gauge_quarter@2x.png 2x" />
+<img class="responsive-img" src="../images/radial_gauge_full-thin.png" srcset="../images/radial_gauge_full-thin@2x.png 2x" />
+<img class="responsive-img" src="../images/radial_gauge_half-thin.png" srcset="../images/radial_gauge_half-thin@2x.png 2x" />
+<img class="responsive-img" src="../images/radial_gauge_quarter-thin.png" srcset="../images/radial_gauge_quarter-thin@2x.png 2x" />
+
```

Comment:
Add a section for range thickness with an explanation of the two variants. Add the necessary images.